docs/docs/policies.mdx

```diff
@@ -385,6 +385,16 @@ for details.
 The `Rule Policy` is a policy that handles conversation parts that follow
 a fixed behavior. Please see [Rules](./rules.mdx) for further information.
 
+You can remove rule restriction and check for contradictions by setting the following
```

Comment:
do we actually want to advertise this? (cc @akelad )

If we add this, we need to first explain what the "restrictions" are.

Reply:
we should document it, yes. But +1 to explaining what the restrictions are

Reply:
the restriction is explained in `rules.mdx`

Reply:
then let's link there